mistralrs/Cargo.toml

```diff
@@ -70,7 +70,11 @@ name = "idefics2"
 required-features = []
 
 [[example]]
-name = "anymoe"
```

Comment:
I think you might have removed `anymoe` and replaced it with `llava` here?

Reply:
ok fixed it